### ISSUE TITLE
Roadmap for official release?

### ISSUE THREAD

**Issue by Aklakan** [afs/jena-rdfconnection#2]
Hi, I assume that you intent to make this project part of the official jena release. Do you have a roadmap for that when it will be available on maven central?

**Comment by afs** [comment]
Yes, that's my hope. Currently, some lower level changes in how auth works as we upgrade from the latest style of httpClient APIs is in the way. However, it might be an idea to provide an auth-less version to get discussion going. As an API whose point is to be solid and stable long term, just dropping it in the codebase and saying "done" is not the way to proceed.

Have you used it at all? Any comments?  Any simple things getting in your way?
